include/oneapi/mkl/dft/detail/descriptor_impl.hpp

```diff
@@ -68,22 +68,22 @@ class descriptor {
     void commit(backend_selector<backend::mklgpu> selector);
 #endif
 
-    dft_values<prec, dom> get_values() {
+    dft_values<prec, dom>& get_values() {
```

Comment:
Shouldn't this return a const reference to avoid bypassing any checks that would be in `set_value`? The method could be made `const` too.

Reply:
With the current design, `dft_values` includes the `changed` set that needs to be updated. If we get rid of `changed` then this should be altered as you suggest.

Reply:
This is now const